core/src/main/java/org/apache/accumulo/core/util/ThriftMessageUtil.java

```diff
@@ -49,7 +50,11 @@ public ThriftMessageUtil(int initialCapacity) {
     // TODO does this make sense? better to push this down to the serialize method (accept the
     // transport as an argument)?
     this.initialCapacity = initialCapacity;
-    this.inputTransport = new TMemoryInputTransport();
+    try {
+      this.inputTransport = new TMemoryInputTransport();
+    } catch (TTransportException e) {
+      throw new RuntimeException(e);
```

Comment:
Looking at the code that calls this, it may be better to throw an IOException here.